Tasks/Singh/Task1/Task1.py.rtf

```diff
@@ -0,0 +1,59 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2639
+\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
+{\colortbl;\red255\green255\blue255;}
+{\*\expandedcolortbl;;}
+\paperw11900\paperh16840\margl1440\margr1440\vieww11520\viewh8400\viewkind0
+\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\pardirnatural\partightenfactor0
+
+\f0\fs24 \cf0 +import math\
++ \
++  print("\uc0\u1042 \u1074 \u1077 \u1076 \u1080 \u1090 \u1077  \u1082 \u1086 \u1101 \u1092 \u1092 \u1080 \u1094 \u1080 \u1077 \u1085 \u1090 \u1099  \u1076 \u1083 \u1103  \u1091 \u1088 \u1072 \u1074 \u1085 \u1077 \u1085 \u1080 \u1103 ")\
++ a = int(input("a = "))\
++  b = int(input("b = "))\
++  c = int(input("c = "))\
++ \
++  discr = b ** 2 - 4 * a * c\
++  print("\uc0\u1044 \u1080 \u1089 \u1082 \u1088 \u1080 \u1084 \u1080 \u1085 \u1072 \u1085 \u1090  D = %.2f" % discr)\
++ \
++  if discr > 0:\
++      x1 = (-b + math.sqrt(discr)) / (2 * a)\
++      x2 = (-b - math.sqrt(discr)) / (2 * a)\
++      print("x1 = %.2f \\nx2 = %.2f" % (x1, x2))\
++  elif discr == 0:\
++      x = -b / (2 * a)\
++      print("x = %.2f" % x)\
++  else:\
```

Comment:
this is a part of another task, I do not need it here, -10 points for the structural mistake